roles/openshift_sdn/files/sdn.yaml

```diff
@@ -86,6 +86,8 @@ spec:
           # Take over network functions on the node
           rm -Rf /etc/cni/net.d/80-openshift-network.conf
           cp -Rf /opt/cni/bin/* /host/opt/cni/bin/
+          cp -f /opt/cni/bin/openshift-sdn /host/opt/cni/bin/openshift-sdn-${OPENSHIFT_SDN_POD}
```

Comment:
hm... but this will cause the 3.11 build to be broken until we commit the corresponding change to origin, won't it?

I think we need to do a 3-step update here; either

1. change openshift-ansible to copy the binary to _both_ the old and the new paths
2. change origin to use the new path
3. change openshift-ansible to copy only to the new path

or

1. change origin to support _both_ the old path and the new path
2. change openshift-ansible to use only the new path (this PR)
3. change origin to support only the new path

And likewise for the 4.x version probably, except with CNO rather than openshift-ansible

Reply:
@danwinship this does copy both. It copies everything in /opt/cni/bin and copies the specific binary in the way that we want it. Look at line 88

Reply:
oh, duh
/lgtm